tasklauncher-cloudfoundry-app-dependencies/pom.xml

```diff
@@ -13,15 +13,15 @@
     <parent>
         <artifactId>spring-cloud-dependencies-parent</artifactId>
         <groupId>org.springframework.cloud</groupId>
-        <version>1.2.1.RELEASE</version>
+        <version>1.2.1.BUILD-SNAPSHOT</version>
```

Comment:
Just curious why we wouldn't go ahead and uses either the 1.3.2.RELEASE or the 1.3.3.BUILD-SNAPSHOT?